presto-native-execution/scripts/setup-adapters.sh

```diff
@@ -28,9 +28,17 @@ function install_jwt_cpp {
   cmake_install -DBUILD_TESTS=OFF -DJWT_BUILD_EXAMPLES=OFF -DJWT_DISABLE_PICOJSON=ON -DJWT_CMAKE_FILES_INSTALL_DIR="${DEPENDENCY_DIR}/jwt-cpp"
 }
 
+function install_prometheus_cpp() {
+  github_checkout jupp0r/prometheus-cpp v1.2.4 --depth 1
+  git submodule init
+  git submodule update
+  cmake_install -DBUILD_SHARED_LIBS=ON -DENABLE_PUSH=OFF -DENABLE_COMPRESSION=OFF
+}
+
 cd "${DEPENDENCY_DIR}" || exit
 
 install_jwt=0
+install_prometheus_cpp=1
```

Comment:
This should be 0. Set it to 1 if no argument is given after line 45.
With this change, if someone calls `setup-adapter.sh jwt`, it will also install the prometheus-cpp one.